SleepEngine/Engine/EngineConfig.h

```diff
@@ -15,4 +15,9 @@ static glm::vec2 const PrimaryWindowSize(1024.f, 1024.f);
 static auto EngineLogger = "engine_logger";
 static auto EngineLoggerPath = "Logs/engine_logger.log";
 
+namespace Resources
+{
+    static auto ImagesTablePath = "Engine/Generated/textures_table.json";
```

Comment:
Static is not needed? And better to make this const

Reply:
1) Static is needed because of: 
![image](https://user-images.githubusercontent.com/24576933/55287371-93c4ab80-53b0-11e9-95b2-1e18911bd933.png)
2) const is redundant because it is a **const char***